drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -103,6 +103,12 @@ public Capabilities getCapabilities(WebDriverConfiguration configuration, boolea
         new ChromeDriverBinaryHandler(capabilities).checkAndSetBinary(performValidations);
 
         ChromeOptions chromeOptions = new ChromeOptions();
+
+        String browser = configuration.getBrowser();
+        if (browser.equals("chromeHeadless") || browser.equals("chromeheadless")) {
```

Comment:
Don't we expect chrome to show up browser window unless it is in headless mode ? 
Also, if a user wishes to use `chrome` instead of `chromeheadless` and still be able to use headless mode that can still in done as per the original way. 
```
<property name="browser">chrome</property>
<property name="chromeArguments">--headless</property> 
```
Please correct me, if my interpretation here is wrong.